lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -138,6 +138,7 @@ def add_credentials(post, command)
 
       def add_transaction_elements(post, type, options)
         transaction = {}
+        transaction[:paymentCountry] = options[:payment_country] || options[:billing_address][:country] if (options[:billing_address] && options[:billing_address][:country])
```

Comment:
I don't _believe_ the additional ` && options[:billing_address][:country]` is necessary on this guard clause since `options[:billing_address][:country]` will evaluate to `nil` as long as the `:billing_address` key exists.